pkg/arvo/app/aqua.hoon

```diff
@@ -511,24 +508,37 @@
     ::      should be deleted now that aqua is capable of managing azimuth state
     ::      internally. Its been left this way for now until all the ph tests
     ::      can be rewritten
-    =/  keys=dawn-event:jael  (dawn who.ae)
+    =/  keys=dawn-event:jael  
+      ?~  keys.ae  *dawn-event:jael
+      (dawn who.ae)
     =.  this  abet-pe:(publish-effect:(pe who.ae) [/ %sleep ~])
     =/  initted
       =<  plow
       %-  push-events:apex:(pe who.ae)
       ^-  (list unix-event)
       :~  [/ %wack 0]  ::  eny
-          [/ %whom who.ae]  ::  eny
-          [//newt/0v1n.2m9vh %born ~]
-          [//behn/0v1n.2m9vh %born ~]
-          :^  //term/1  %boot  &
+          ::  [/ %verb `|]  possible verb
+          :^  /  %wyrd  [%urbit /test] :: negotiate XX what is .non and .rev
+          ^-  (list (pair term @))
+          :~  zuse+zuse
+              lull+lull
+              arvo+arvo
+              hoon+hoon-version
+              nock+4
+          ==
+          [/ %whom who.ae]  ::  who
+          -.kernel-ova.pil  :: load compiler
+          +<:kernel-ova.pil :: load arvo XX how to work with solid?
```

Comment:
I think we can just weld in `kernel-ova.pil` instead of implicitly asserting it has two events.  I think that should be all that's needed to support solid pills.  `userspace.ova` below should also be welded.